### PR TITLE
Add "Remove White spaces from cached views."

### PR DESCRIPTION
**Pull request by Gami-404**
Add "Remove White spaces from compiled views."
i remove white space from HTML which in compiled views  files then it's rendered html without  any white space

### CODE REVIEW

File: src/Illuminate/View/Compilers/BladeCompiler.php [hunk 1]
```diff
@@ -136,12 +136,49 @@ public function compile($path = null)
                 $contents .= "<?php /**PATH {$this->getPath()} ENDPATH**/ ?>";
             }
 
+            if ($this->app['config']['view.removeWhitespace'] == true) {
```
Comment (browner12) [inline src/Illuminate/View/Compilers/BladeCompiler.php]: please use underscore casing on "removeWhitespace" for consistency with existing configuration.

I believe we'll also want to use a strict equality here, so `===`, so not all truthy values are accepted.

**Comment by browner12** [comment]
What is the payoff with this PR?

Is it for performance reasons, and if so are the improvements significant?

**Comment by laurencei** [comment]
Doesn't gzip effectively handle this for us? I dont think you get much, if any, benefit stripping white spaces on a compressed page anyway.

And for the increased risk of bugs and formatting errors on legitimate sections - seems like a risk to me... 🤷‍♂

**Comment by Gami-404** [comment]
> What is the payoff with this PR?
    Reduce render content size by remove white-spaces between html content
> Is it for performance reasons, and if so are the improvements significant?
   Yes , it for performance

**Comment by Gami-404** [comment]
@laurencei 
it's good answer ,  i'am sorry to miss this part   
@browner12 @laurencei   Thanks you

**Pull request closed without merging**